subprojects/docs/src/docs/release/notes.md

```diff
@@ -1,12 +1,15 @@
 The Gradle team is pleased to announce Gradle 4.6.
 
-First and foremost, this release of Gradle features extensive improvements to dependency management. You can now [declare dependency constraints for transitive dependencies](#dependency-constraints-for-transitive-dependencies) and avoid problems caused by oft-hidden upstream dependency changes. 
+This release of Gradle includes built-in support for JUnit Platform and the JUnit Jupiter/Vintage Engine, also known as [JUnit 5 support](#junit-5-support).
+You can use the new filtering and engines functionality in JUnit 5 using the examples provided below and in the documentation.
```

Comment:
Probably can join this sentence to the one above in the same paragraph

Reply:
It is already part of the same paragraph. Just applying a sentence per line formatting.